playground/bicep/BicepSample.AppHost/aspire-manifest.json

```diff
@@ -23,7 +23,7 @@
     },
     "test0": {
       "type": "azure.bicep.v0",
-      "path": "../../../../../../Users/davifowl/AppData/Local/Temp/tmppj1k21.tmp.bicep"
+      "path": "../../../../../Users/midenn/AppData/Local/Temp/tmpvqsa5k.tmp.bicep"
```

Comment:
Yikes 😄

Reply:
Yep. I've got an issue for this I think: https://github.com/dotnet/aspire/issues/3273

Reply:
OK I integrated the change to fix the path in thsi PR too.